projects/tinkerbell/rufio/docker/linux/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as ipmitool-builder
+
+WORKDIR /
+
+RUN yum install -y \
```

Comment:
I mentioned this on another PR, in another repo 😆  - why build ipmitool instead of `yum install ipmitool`?

Reply:
If the version in yum is workable, that would def be preferable. If not, this build looks reasonable, I think its basically the same from one of the old tink builds?

Reply:
`yum` has 1.18 - it depends what branch is being tracked though which is typically dictated, out of the box, by the distro version. I don't know what the base image is.

Reply:
the base images are all al2. the current version in al2 is `ipmitool.x86_64 0:1.8.18-9.amzn2`

Reply:
@pokearu It should do. Give it a go 😄

Reply:
I tested it out and seems to work fine 👍 I have updated it to install from yum and will continue to test and monitor.